_includes/code-of-conduct-short.html

```diff
@@ -0,0 +1,12 @@
+<section id="code_of_conduct">
+    <h2>Code of Conduct</h2>
+
+<p>satRday is dedicated to providing a harassment-free and inclusive conference experience for all in attendance regardless of, but not limited to, gender, sexual orientation, disabilities, physical attributes, age, ethnicity, social standing, religion or political affiliation.</p>
+
+<p>We do not tolerate harassment of participants (including organisers and vendors) in any form. Sexual innuendos and imagery are not appropriate for any conference venue, including presentations.</p>
+
+<p>Anyone violating these rules may be given warning or expelled from the conference (without a refund) at the discretion of the conference organisers.</p> 
+
+<p>Our code of conduct/anti-harassment policy can be found <a href="http://capetown2018.satrdays.org/code-of-conduct.md"> here </a> </p>
```

Comment:
Given that we want to link to the full code of conduct from the website, I think that it's worth converting this to a static html page instead of just markdown so that it can be rendered with some basic styling. I'm happy to do this if you want me to (just let me know and I'll make a new pull request against your branch to update it), otherwise can you give it a bash and update this pull request?

Reply:
Hi Jon,

Thanx for your feedback. I'll give this a bash :)